src/iso.js

```diff
@@ -1,5 +1,14 @@
 const KEY_NAME = 'data-iso-key'
 
+const rLt = /</g
+const rGt = />/g
+const rLte = /&lt;/g
+const rGte = /&gt;/g
+const rEnc = /[<>]/
+const rDec = /&lt;|&gt;/
+
+const toString = val => val ? String(val) : ''
```

Comment:
using `toString` like this is an unrecoverable runtime error in some older browsers - it'd be better to name it something different (something that's not a preexisting global or Object.prototype key).